apps/wallet/src/ui/app/shared/image-icon/index.tsx

```diff
@@ -10,17 +10,22 @@ const imageStyle = cva(['text-white capitalize overflow-hidden bg-gray-40  shrin
 			sm: 'w-6 h-6 font-medium text-subtitleSmallExtra',
 			md: 'w-7.5 h-7.5 font-medium text-body',
 			lg: 'w-10 h-10 font-medium text-heading4',
-			xl: 'w-15 h-15 font-medium text-heading4',
+			xl: 'w-12.5 h-12.5 font-medium text-heading4',
+			xxl: 'w-15 h-15 font-medium text-heading4',
 		},
 		circle: {
 			true: 'rounded-full',
 			false: 'rounded-md',
 		},
+		background: {
+			none: 'bg-transparent',
+		},
 	},
 
 	defaultVariants: {
 		circle: false,
 		size: 'md',
+		background: 'none',
```

Comment:
Does this even need to be a variant? Could you just make this part of the default classes?

Reply:
updated